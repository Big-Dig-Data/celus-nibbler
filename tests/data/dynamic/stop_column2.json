{
  "kind": "non_counter.metric_based",
  "version": 1,
  "parser_name": "stop_column2",
  "data_format": {
    "name": "stop_format"
  },
  "platforms": [
    "Platform1",
    "Platform2"
  ],
  "metrics_to_skip": [],
  "titles_to_skip": [],
  "dimensions_to_skip": {},
  "metric_aliases": [],
  "dimension_aliases": [],
  "heuristics": {
    "conds": [
      {
        "pattern": "^Month$",
        "coord": {
          "row": 0,
          "col": 0
        },
        "kind": "regex"
      }
    ],
    "kind": "and"
  },
  "areas": [
    {
      "data_headers": {
        "roles": [
            {
                "source": {
                    "coord": {
                      "row": 0,
                      "col": 1
                    },
                    "direction": "right"
                },
                "role": "metric"
            }
        ],
        "data_direction": "down",
        "data_cells": {
          "coord": {
            "row": 1,
            "col": 1
          },
          "direction": "right"
        },
        "skip_condition": {
            "kind": "or",
            "conds": [
                {
                    "kind": "regex",
                    "pattern": "^Metric3$"
                },
                {
                    "kind": "index",
                    "min": 1,
                    "max": 1
                }
            ]
        }
      },
      "dates": {
          "source": {
              "coord": {
                  "row": 1,
                  "col": 0
              },
              "direction": "down"
          },
          "role": "date"
      },
      "titles": null,
      "title_ids": [],
      "dimensions": [],
      "kind": "non_counter.metric_based"
    }
  ]
}
