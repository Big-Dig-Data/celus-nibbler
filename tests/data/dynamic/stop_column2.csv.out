"2020-01-01","2020-01-31","","","Metric1","","","11"
"2020-02-01","2020-02-29","","","Metric1","","","12"
"2020-03-01","2020-03-31","","","Metric1","","","13"
